theia-plugin/yarn.lock

```diff
@@ -0,0 +1,1285 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
`theia-plugin` has both lock files comitted, remove the `package-lock.json` from here

Reply:
Lerna will run `npm install` here, so wouldn't it be better to remove the `yarn.lock` file?

Reply:
As Theia uses yarn I'd have guessed that we don't need the package-lock. If dependencies are installed via npm where does the yarn.lock file come from then?

Reply:
Yes the plugin used yarn initially, but we replaced it with npm and it worked fine. And running the plugin for the first time created this `yarn.lock`. Until now there was no downside for using `npm` over `yarn` here. What is your suggestion?

Reply:
I removed the `yarn.lock` for now.